src/main/java/net/dv8tion/jda/api/managers/ChannelManager.java

```diff
@@ -405,7 +405,7 @@ default ChannelManager sync()
      * <p>A channel slowmode <b>must not</b> be negative nor greater than {@link net.dv8tion.jda.api.entities.TextChannel#MAX_SLOWMODE TextChannel.MAX_SLOWMODE}!
      * <br><b>This is only available to {@link net.dv8tion.jda.api.entities.TextChannel TextChannels}</b>
      *
-     * <p>Note bots are unaffected by this.
+     * <p>Note: Bots are unaffected to this.
```

Comment:
I messed up my words in the previously provided suggestion

```suggestion
     * <p>Note: Bots are unaffected by this.
```